src/Data/Textile/Simulator.elm

```diff
@@ -48,6 +49,8 @@ encode v =
         [ ( "inputs", Inputs.encode v.inputs )
         , ( "lifeCycle", LifeCycle.encode v.lifeCycle )
         , ( "impacts", Impact.encode v.impacts )
+
+        -- TODO: encode complementsImpacts?
```

Comment:
Do we need it encoded? Where would it be used? In the API results? In a debug view like we have in the food simulator?

Reply:
Hence the question mark… It may be useful as an API detailed response indeed, so I'll implement it